onnxscript/function_libs/torch_aten/ops/linalg.py

```diff
@@ -0,0 +1,246 @@
+    raise NotImplementedError()
+
+
+def aten_linalg_eigh(self: TensorType, UPLO: str = "L") -> tuple[TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_eigvals(self: TensorType) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_eigvalsh(self: TensorType, UPLO: str = "L") -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_householder_product(input: TensorType, tau: TensorType) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_inv(A: TensorType) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_inv_ex(
+    A: TensorType, *, check_errors: bool = False
+) -> tuple[TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_ldl_factor(
+    self: TensorType, *, hermitian: bool = False
+) -> tuple[TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_ldl_factor_ex(
+    self: TensorType, *, hermitian: bool = False, check_errors: bool = False
+) -> tuple[TensorType, TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_ldl_solve(
+    LD: TensorType, pivots: TensorType, B: TensorType, *, hermitian: bool = False
+) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_lstsq(
+    self: TensorType,
+    b: TensorType,
+    rcond: Optional[float] = None,
+    *,
+    driver: Optional[str] = None,
+) -> tuple[TensorType, TensorType, TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_lu(
+    A: TensorType, *, pivot: bool = True
+) -> tuple[TensorType, TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_lu_factor(
+    A: TensorType, *, pivot: bool = True
+) -> tuple[TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_lu_factor_ex(
+    A: TensorType, *, pivot: bool = True, check_errors: bool = False
+) -> tuple[TensorType, TensorType, TensorType]:
+    raise NotImplementedError()
+
+
+def aten_linalg_lu_solve(
+    LU: TensorType,
+    pivots: TensorType,
+    B: TensorType,
+    *,
+    left: bool = True,
+    adjoint: bool = False,
+) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_matmul(self: TensorType, other: TensorType) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_matrix_exp(self: TensorType) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_matrix_norm(
+    self: TensorType,
+    ord: float,
+    dim: Sequence[int] = (-2, -1),
+    keepdim: bool = False,
+    *,
+    dtype: Optional[int] = None,
+) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_matrix_power(self: TensorType, n: int) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_matrix_rank(
+    self: TensorType, tol: float, hermitian: bool = False
+) -> TensorType:
+    raise NotImplementedError()
+
+
+def aten_linalg_multi_dot(tensors: TensorType[...]) -> TensorType:
```

Comment:
`TensorType[...]` does not likely do what you intended here. Is this supposed to be a variadic operator?

Reply:
I was hoping to write “a tensor of any type and any rank”. What would be the right way to do it?

Reply:
This actually should be a list of tensors. I will fix